src/test/java/io/jenkins/plugins/checks/steps/PublishChecksStepITest.java

```diff
@@ -26,7 +39,17 @@ public void shouldPublishChecksWhenUsingPipeline() throws IOException {
                 + "text: 'Pipeline support for checks', status: 'IN_PROGRESS', conclusion: 'NONE'"));
 
         assertThat(JenkinsRule.getLog(buildSuccessfully(job)))
-                .contains("[Pipeline] publishChecks")
-                .contains("[Checks API] No suitable checks publisher found.");
+                .contains("[Pipeline] publishChecks");
+
+        assertThat(PUBLISHER_FACTORY.getPublishedChecks().size()).isEqualTo(1);
+
+        ChecksDetails details = PUBLISHER_FACTORY.getPublishedChecks().get(0);
+
+        assertThat(details.getName().get()).isEqualTo("customized-check");
```

Comment:
Typically, one would use in AssertJ:
```suggestion
       assertThat(details.getName()).isPresent().get().isEqualTo("customized-check");
```
This will also not require the annotation.

Reply:
Oh, neat! Thanks.